src/spec/page-objects/settings-page.js

```diff
@@ -0,0 +1,23 @@
+class SettingsPage {
+  constructor() {
+    this._url = '/#/settings/domain-manager';
+    this._togglerDomainInputButton = $('.domain--input-toggler');
+    this._confirmAddDomainButton = $('.domain--input-container button');
+    this._domainInputContainer = $('.domain--input-container');
+  }
+
+  getUrl(){
+    return this._url;
+  }
+  clickInputToggler(){
+    return this._togglerDomainInputButton.click();
+  }

```

Comment:
What do you think about to reduce the scope of this PR to a ReadOnly list of domains? after that you could add a new PR for Add, Remove, etc